crypto/transactions/transaction.py

```diff
@@ -126,79 +101,75 @@ def parse_signatures(self, serialized, start_offset):
         Returns:
             None: methods returns nothing
         """
+        signature_end_offset = start_offset + (64 * 2)
```

Comment:
this needs an additional check, as it currently won't work when a multisig wallet is sending a transaction as that only contains a `signatures` property and no `signature`